src/helpers/database.py

```diff
@@ -97,22 +97,29 @@ def write_fees(
         token_address: str,
         fee_amount: float,
         fee_type: str,
+        recipient: str,
     ):
         """Function attempts to write price data to the table."""
         tx_hash_bytes = bytes.fromhex(tx_hash[2:])
         token_address_bytes = bytes.fromhex(token_address[2:])
         order_uid_bytes = bytes.fromhex(order_uid[2:])
+
         query = read_sql_file("src/sql/insert_fee.sql")
+        final_recipient = None
+        if recipient != "":
+            final_recipient = bytes.fromhex(recipient[2:])
+
         self.execute_and_commit(
             query,
             {
-                "chain_name": self.chain_name,
+                "chain_name": chain_name,
```

Comment:
It would be more consistent with the other functions to remove `chain_name` from the arguments and use `self.chain_name` here.